casepro/cases/models.py

```diff
@@ -103,6 +103,16 @@ def __str__(self):
         return self.name
 
 
+class SystemUser(User):
+
+    @classmethod
+    def get_or_create(cls):
+        systemUsers = cls.objects.filter()
+        if len(systemUsers) > 0:
```

Comment:
We could still use `count` or `exists` here, so that the database returns either just a number or a boolean, instead of all the details of all of the rows.